### PR TITLE
Fix `python-dotenv` dependency for Python 2 compatibility

### LINKED ISSUE #181: Dependency python-dotenv breaks Python 2 support
The latest release of `python-dotenv` removes support for Python 2. The dependency spelled out in `setup.py` should be restricted to a version < 0.19.

### PR DESCRIPTION
**Pull request by dataflake**
Fixes #181

### CODE REVIEW

**Comment by dataflake** [comment]
@jenkins-plone-org please run jobs

File: setup.py [hunk 1]
```diff
@@ -54,7 +54,7 @@
         "ZEO",
         "waitress >= 1.2.0",
         "Paste",
-        "python-dotenv",
+        "python-dotenv < 0.19",
```
Comment (jensens) [inline setup.py:57]: comment why
```suggestion
        "python-dotenv < 0.19",  # python 2 support
```

**Comment by mauritsvanrees** [comment]
Can't we restrict it only on Python 2, so people on Python 3 can use the latest if they want?
Untested, but something like this:

```
'python-dotenv < 0.19; python_version<"3"',
'python-dotenv; python_version>="3"',
```

**Comment by dataflake** [comment]
@jenkins-plone-org please run jobs

**Review by jensens — APPROVED** [review]

**Pull request merged**